ipa-core/Cargo.toml

```diff
@@ -116,7 +116,7 @@ hyper = { version = "0.14.28", optional = true, features = [
     "stream",
     "runtime",
 ] }
-hyper-rustls = { version = "0.25.0", optional = true, features = ["http2"] }
+hyper-rustls = { git = "https://github.com/cberkhoff/hyper-rustls/", branch = "rustls-0.23", version = "0.25.1", optional = true, features = ["http2"] }
```

Comment:
I am a bit more nervous about forking `hyper-rustls` than `axum-server`. Is it also a temporary measure to help us get to the end state?

Reply:
Correct, this is just a temporary step. It's just easier and faster for me to fully update the TLS stack and be done with that, even though that's not how hyper versions were released. The change I made on hyper-rustls is very minor, just a version bump on the TLS stack. To give some insights, my update plan for IPA is as follows:

1. Update Rustls to 0.23 - This change.
2. Update to Axum 0.6.
3. Update to Axum 0.7 - This PR will update many dependencies, hence I'm trying to do carve smaller changes (like 1 and 2). With this change we should be able to stop using my custom versions.

I'll add a comment in the code explaining this is just a temporary step.